MAVProxy/modules/mavproxy_link.py

```diff
@@ -45,7 +45,8 @@ def __init__(self, mpstate):
                           'add (SERIALPORT)',
                           'attributes (LINK) (ATTRIBUTES)',
                           'remove (LINKS)',
-                          'hl (HLSTATE)'])
+                          'hl (HLSTATE)',
+                          'dataratelogging (DLSTATE)'])
```

Comment:
Can we reformat this a little so that when we add a line we don't modify two?

Reply:
Done!